docs/best-practices/api-design.md

```diff
@@ -180,7 +180,7 @@ If the server cannot match any of the media type(s) listed, it should return HTT
 
 ### GET methods
 
-A successful GET method typically returns HTTP status code 200 (OK). If the resource cannot be found, the method should return 404 (Not Found).
+A successful GET method typically returns HTTP status code 200 (OK). If the resource cannot be found, the method should return 404 (Not Found). If the request was fulfilled but there is no data to return, then it should return 204 (No-Content).
```

Comment:
`POST` and `PUT` it's pretty common for a `204`, since it can basically act as an _ack_ for an operation.  While it's for sure possible in a `GET`, it's not all that common.  Maybe you can provide a bit more of a detailed use case where a `GET` would be expected to return no body in an otherwise found resource.  If we can expand this example a bit, it would make this more interesting to add.

Reply:
@ckittel @EdPrice-MSFT Hey, I have just added an example

Reply:
@EdPrice-MSFT -- do you mind taking a quick _edit_ pass on @rodrigoleitetr's [latest commit](https://github.com/MicrosoftDocs/architecture-center/pull/3274/commits/6e402b9db124989bd9f74ee27128cf289094cc63) to align with the suggestion above a bit more.  Then I think we'll be good to merge.

Reply:
I went ahead and made a quick tweak based on @rodrigoleitetr's example.